dolphinscheduler-alert/src/main/java/org/apache/dolphinscheduler/alert/AlertServer.java

```diff
@@ -60,20 +63,17 @@ public void start(){
         logger.info("alert server ready start ");
         while (Stopper.isRunning()){
             try {
-                Thread.sleep(Constants.ALERT_SCAN_INTERVEL);
+                Thread.sleep(Constants.ALERT_SCAN_INTERVAL);
             } catch (InterruptedException e) {
                 logger.error(e.getMessage(),e);
             }
             List<Alert> alerts = alertDao.listWaitExecutionAlert();
             alertSender = new AlertSender(alerts, alertDao);
             alertSender.run();
         }
-    }
 
-
-    public static void main(String[] args){
-        AlertServer alertServer = AlertServer.getInstance();
```

Comment:
if commented, how to start alert server ?

Reply:
I have the same question,why remove the main method?AlertServer now is a independent service